src/config/animations.js

```diff
@@ -0,0 +1,2 @@
+export const speed = 100;
+export const friction = 10;
```

Comment:
😅  Totally my bad, did this quickly completely without testing. Should be fixed now, could you test again?